### DISCUSSION TITLE
How to use sass darken function with var(--pico-color)?

### DISCUSSION THREAD

**Discussion by infografik** [picocss/pico#524] [Help]
darken(var(--pico-color), 20%) -> Error: $color: var(--pico-color) is not a color.
darken(red, 20%) -> works correctly

**Comment by dev-willis** [comment]
I don't think it can be done. Sass functions run at compile time; CSS custom properties are only available at run time.

**Reply by infografik** [reply]
Exactly. It would be nice to have colors set in a different way to simplify the ability to fine-tune the page colors:
$xcolor: #cb4459;
$ycolor: #44c522;
.darken-element {
     background: darken($xcolor,20%);
}
.lighten-element {
     background: lighten($xcolor,20%);
}
.mixed-element {
      background: mix($xcolor,$ycolor,20%)
}
This offers great opportunities.
